CONTRIBUTING.md

```diff
@@ -1,103 +1,118 @@
-# Wandb library
-
-## Get the code / library
-
-Checkout from github:
+# Contributing to `wandb`
+
+We at Weights & Biases ❤️ open source and welcome contributions from the community!
+This guide discusses the development workflow and the internals of the `wandb` client library.
+
```

Comment:
I think it would be great to add table of content here, what do you think?

Reply:
Done